dotnet/src/Functions/Functions.OpenAPI/Model/RestApiOperation.cs

```diff
@@ -177,14 +177,14 @@ public Uri BuildOperationUrl(IDictionary<string, string?> arguments, Uri? server
     /// <param name="path">Operation path to replace parameters in.</param>
     /// <param name="arguments">Arguments to replace parameters by.</param>
     /// <returns>Path with replaced parameters</returns>
-    private string ReplacePathParameters(string path, IDictionary<string, string?> arguments)
+    private string ReplacePathParameters(string path, IDictionary<string, string> arguments)
     {
         string ReplaceParameter(Match match)
         {
             var parameterName = match.Groups[1].Value;
 
             //A try to find parameter value in arguments
-            if (arguments.TryGetValue(parameterName, out string? value) && value is not null)
+            if (arguments.TryGetValue(parameterName, out string value) && value is not null)
```

Comment:
Same question for all of these.

(If the arguments comes in via a public API where we haven't validated non-nullness of the arguments, then it's still possible they're null.)